docs/manual-EN/1.overview/3.design-and-architecture/2.storage-design.md

```diff
@@ -74,69 +73,77 @@ Pict.5 In-key format
 
 ---
 
-The properties of vertices and edges are stored in the values of the corresponding kv pairs after being encoded. As a strong typed database, **Nebula Graph** needs to get the schema information in the Meta Service before decoding. Besides, when properties are encoded, schema version is added to support online schema change.
+Besides the key part above, the value part is the encoded properties (of a vertex or an edge). As a strong typed database, **Nebula Graph** get the schema information from the Meta Service before encoding/decoding. And schema versioning are also considered to support alter schema.
```

Comment:
**Nebula Graph** gets
And schema versioning is also considered to support the schema alterting.